src/pudl/transform/eia.py

```diff
@@ -814,6 +814,17 @@ def _restrict_years(df,
     return df
 
 
+# def _add_eia_epacems_crosswalk(eia_transformed_dfs):
```

Comment:
can you get rid of this lingering code here now that it is integrated into `etl.py`??

Reply:
yes! thanks for spotting that.